docs/resources/eks_cluster.md

```diff
@@ -680,7 +764,7 @@ resource "rafay_eks_cluster" "eks-cluster-1" {
 
 ***Required***
 
-- `attach_policy` - (Block List, Max: 1) Holds a policy document to attach to the service account.
+- `attach_policy` - (String) Holds a policy document to attach to the service account in json string format.
```

Comment:
@gopikrishna-rafay why is type changing for existing field ?

Reply:
@robbierafay The actual supported type is string, this was a mistake is existing documentation

Reply:
Thanks @gopikrishna-rafay